OpenRA.Mods.Common/Widgets/Logic/AssetBrowserLogic.cs

```diff
@@ -239,7 +274,16 @@ public AssetBrowserLogic(Widget widget, Action onExit, ModData modData, WorldRen
 						SelectPreviousFrame();
 				};
 
-				prevButton.IsVisible = () => !isVideoLoaded;
+				prevButton.IsVisible = () => !isVideoLoaded && currentSoundFormat == null;
+			}
+
+			var spriteScaleSlider = panel.GetOrNull<SliderWidget>("SPRITE_SCALE_SLIDER");
+			if (spriteScaleSlider != null)
+			{
+				spriteScaleSlider.OnChange += x => spriteScale = x;
+				spriteScaleSlider.GetValue = () => spriteScale;
+				spriteScaleSlider.IsVisible = () => !isVideoLoaded && currentSoundFormat == null;
+				panel.GetOrNull<LabelWidget>("SPRITE_SCALE").IsVisible = () => !isVideoLoaded && currentSoundFormat == null;
```

Comment:
Icons don't get a scale slider:

![Screenshot_20210917_203506](https://user-images.githubusercontent.com/756669/133837634-c2b716c8-e1c8-47cf-bdf2-9c6c893655e1.png)

This could have been fixed by https://github.com/OpenRA/OpenRA/pull/19498.